### PR TITLE
Fixed issue #38

### PR DESCRIPTION
**Pull request by jmsether**
Implemented image grid detection to stop wrong aspect ratio scaling. Issue #38

### CODE REVIEW

File: iz_helpers/video.py [hunk 1]
```diff
@@ -13,9 +19,19 @@ def write_video(file_path, frames, fps, reversed=True, start_frame_dupe_amount=1
     if reversed == True:
         frames = frames[::-1]
 
-    # Get dimensions of the first frames, all subsequent has to be same sized
```
Comment (GeorgLegato) [inline iz_helpers/video.py:16]: please keep the assert, even more, should output frame[0].size and current invalid frame.size, to get a hint about is it a roundness error +-1 px or do we mess up completeley

**Review by GeorgLegato — COMMENTED** [review]
@v8hid we should first PR into develop branch

**Comment by jmsether** [comment]
The assert should not be needed as there should be no way for frames of different sizes to make it through to ffmpeg anymore.

**Comment by GeorgLegato** [comment]
@v8hid will you approve? I dont see only code change due to mask included in output image. but you were on this topic

**Comment by v8hid** [comment]
@GeorgLegato It's Ok I will test it to make sure everything is fine, the assert is not needed `imageio` has one inside.
@jmsether fix the comments and PR to developed instead.

**Pull request closed without merging**